core/jvm/src/main/scala/zio/Signals.scala

```diff
@@ -0,0 +1,15 @@
+package zio
+
+import sun.misc.{Signal, SignalHandler}
+import zio.console._
+
+object Signals {
+  def fiberDump: SignalHandler =
+    Signal.handle(new Signal("USR1"), (sig: Signal) => {
+      for {
```

Comment:
For this to actually print, you'll need the runtime here and an `unsafeRun` on the resulting effect.